shared/actions/json/chat2.json

```diff
@@ -198,6 +198,7 @@
     },
     "leaveConversation": {
       "conversationIDKey": "Types.ConversationIDKey",
+      "navigateToInbox?": "boolean",
```

Comment:
i think we want the opposite. you always want to navigate to the inbox unless we opt out i think?

Reply:
k we can keep this either way. depends on if its more common to do this within the context of a conversation or externally (manage channels) up to you

Reply:
yup!